net.kvirc.KVIrc.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-icon: kvirc
+rename-desktop-file: kvirc.desktop
+command: kvirc
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=xdg-download
+  - --persist=$XDG_CONFIG_HOME/KVIrc
```

Comment:
As mentioned in the previous PR, this is not going to work. It should be `--persist=.config/KVIrc`.

Reply:
Okay. Thanks.

Changes in the way